modules/social_features/social_event/src/Service/SocialEventEnrollService.php

```diff
@@ -34,28 +34,9 @@ public function __construct(ConfigFactoryInterface $config_factory) {
    * {@inheritdoc}
    */
   public function isEnabled(NodeInterface $node) {
-    if (
-      $this->eventSettings->get('disable_event_enroll') ||
-      $node->bundle() !== 'event' ||
-      !$node->hasField('field_event_enroll') ||
-      (!$node->get('field_event_enroll')->isEmpty() && !(bool) $node->get('field_event_enroll')->getString())
-    ) {
-      return FALSE;
-    }
-
-    $was_not_changed = $node->get('field_event_enroll')->isEmpty();
-    $is_enabled = (bool) $node->get('field_event_enroll')->getString();
-
-    // Make an exception for the invite enroll method.
-    // This doesn't allow people to enroll themselves, but get invited.
-    if (
-      !$node->get('field_enroll_method')->isEmpty() &&
-      (int) $node->get('field_enroll_method')->getString() === EventEnrollmentInterface::ENROLL_METHOD_INVITE
-    ) {
-      $is_enabled = TRUE;
-    }
-
-    return $was_not_changed || $is_enabled;
+    @trigger_error(__METHOD__ . '() is deprecated in social:11.4.0 and is removed from social:12.0.0. Use bundled node object itself `$event->isEnrollmentEnabled()` instead. See https://www.drupal.org/project/social/issues/3306568', E_USER_DEPRECATED);
```

Comment:
@nechai Please change version from 11.4 to 11.5

Reply:
Added